src/foam/nanos/dig/rules.jrl

```diff
@@ -0,0 +1,28 @@
+p({
+  "class":"foam.nanos.ruler.Rule",
+  "id":"set-dug-rule-acting-user-rule",
+  "name":"Set DUG Rules acting user if it's blank",
```

Comment:
EnsurePropertyOnCreateRule almost can accomplish the same thing here except the `propValue` has to be provided. For your case, we might need a way to get the propValue from another property.

Eg.
```
p({
  "class":"foam.nanos.auth.ruler.EnsurePropertyOnCreateRule",
  "id":"set-dug-rule-acting-user-rule",
  "name":"Set DUG Rules acting user if it's blank",
  "priority": 100,
  "enabled": true,
  "ruleGroup": "DUG",
  "daoKey": "localRuleDAO",
  "targetClass": "foam.nano.dig.DUGRule",
  "propName": "actingUser",
  "propValueFrom": "createdBy",
  "lifecycleState": 1
})
```

Reply:
I can look into implementing that option

Reply:
@lchanmann implemented that feature in teh rule